### DISCUSSION TITLE
Muscle Group

### DISCUSSION THREAD

**Discussion by saitauthao** [LiamMorrow/LiftLog#835] [Ideas]
Adding muscle group to each exercise that is being worked on, so you can visual see which muscle group you are targeting. e.g.(Bench Press, Primary Muscle Groups: Chest, Shoulders, Triceps) 

![received_1144387370355911](https://github.com/user-attachments/assets/70c57956-7c39-4511-a9a7-7cd27e615395)

**Comment by LiamMorrow** [comment]
Hi @saitauthao, thanks for the suggestion! I can definitely see this being part of a larger exercise overhaul, and could be a nice tool for determining if your split is under/over working specific muscle groups.  I'll keep it in mind
